pkg/engine/lifecycletest/pulumi_test.go

```diff
@@ -3558,8 +3558,8 @@ func TestProtect(t *testing.T) {
 	assert.Equal(t, 0, deleteCounter)
 
 	// Run a new update which will cause a delete, we still shouldn't see a provider delete
-	expectedMessage = "<{%reset%}>unable to delete resource \"urn:pulumi:test::test::pkgA:m:typA::resA\"\n" +
-		"as it is currently marked for protection. To unprotect the resource, either remove the `protect` flag " +
+	expectedMessage = "<{%reset%}>resource \"urn:pulumi:test::test::pkgA:m:typA::resA\" is marked for protection\n" +
+		"so it cannot be deleted. To unprotect the resource, either remove the `protect` flag " +
```

Comment:
I like the direction, but I think that we can be a bit more succinct. What do you think of this?

```suggestion
	expectedMessage = "<{%reset%}>resource \"urn:pulumi:test::test::pkgA:m:typA::resA\" cannot be deleted because it is protected.\n" +
		"To unprotect the resource, either remove the `protect` flag " +
```

Reply:
It sounds a little blunter. I like it.